frontend/src/component/changeRequest/ChangeRequest.test.tsx

```diff
@@ -298,7 +291,8 @@ test('add flag change to pending change request', async () => {
 
     await verifyBannerForPendingChangeRequest();
 
-    await changeFlag('production');
+    const flag = screen.getByLabelText('production');
+    fireEvent.click(flag);
```

Comment:
we don't have buttons on the left to toggle if flag is enabled/disabled in an environment

Reply:
Sure is! We could even use `getByRole` instead (`getByRole('button', {name: 'production'}`) to follow [React Testing Library's priority guidelines](https://testing-library.com/docs/queries/about/#priority). I think that makes it clearer what we're doing too. (and maybe rename the variable to `environmentToggle` or something).

But overall, nah. That's fine. This is good.